app/views/deployments/show.html.erb

```diff
@@ -0,0 +1,78 @@
+<div class="mt-4">
+  <%= render @deployment%>
+</div>
+
+<div class="bg-white shadow overflow-hidden sm:rounded-lg">
+  <div class="px-4 py-5 sm:px-6">
+    <h3 class="text-lg leading-6 font-medium text-gray-900">
+      Argo Deployment Stats
+    </h3>
+  </div>
+
+	<% if @response.successful? %>
+  <div class="border-t border-gray-200 px-4 py-5 sm:p-0">
+    <dl class="sm:divide-y sm:divide-gray-200">
+      <div class="py-4 sm:py-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-6">
+				<dt class="text-sm font-medium text-gray-500">
+				  App Health
+				</dt>
+				<dd class="mt-1 text-sm text-gray-900 sm:mt-0 sm:col-span-2">
+				  <%= @response.app_health %><br/>
+					Message: <%= @response.app_health_message || "none"%>
+				</dd>
+      </div>
+    </dl>
+  </div>
+  <div class="border-t border-gray-200 px-4 py-5 sm:p-0">
+    <dl class="sm:divide-y sm:divide-gray-200">
+      <div class="py-4 sm:py-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-6">
+				<dt class="text-sm font-medium text-gray-500">
+				  Current Commit Info
+				</dt>
+				<dd class="mt-1 text-sm text-gray-900 sm:mt-0 sm:col-span-2">
+					Status: <%= @response.current_git_status %><br/>
+					Git Revision: <%= @response.current_git_revision %>
+				</dd>
+      </div>
+    </dl>
+  </div>
+  <div class="border-t border-gray-200 px-4 py-5 sm:p-0">
+    <dl class="sm:divide-y sm:divide-gray-200">
+      <div class="py-4 sm:py-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-6">
+				<dt class="text-sm font-medium text-gray-500">
+				  Previous Commit Info
+				</dt>
+				<dd class="mt-1 text-sm text-gray-900 sm:mt-0 sm:col-span-2">
+					Deployed at: <%= @response.previous_git_deployed_at %><br/>
+					Git Revision: <%= @response.previous_git_revision %>
+				</dd>
+      </div>
+    </dl>
+  </div>
+	<% else %>
+	  <div class="border-t border-gray-200 px-4 py-5 sm:p-0">
+	    <dl class="sm:divide-y sm:divide-gray-200">
+	      <div class="py-4 sm:py-5 sm:grid sm:grid-cols-3 sm:gap-4 sm:px-6">
+					<dt class="text-sm font-medium text-gray-500">
+						<%= @response.status %> - <%= @response.error_message %>
+					</dt>
+					<dd class="mt-1 text-sm text-gray-900 sm:mt-0 sm:col-span-2">
+						Error: Something went wrong with the Argo API call, please try again
+					</dd>
+	      </div>
+	    </dl>
+	  </div>
+	<% end %>
+</div>
```

Comment:
Could you fix the indentation here?

Reply:
Holy cow this is ugly lol! Must be my editor

Reply:
Indentation changes here: https://github.com/department-of-veterans-affairs/platform-console-api/pull/149/commits/c358fccd80f5f79f8743dde964abc0c3a487dacc